src/components/blocks/software/Code/MultilineCodeContent.tsx

```diff
@@ -1,8 +1,13 @@
-import React, { useLayoutEffect, useRef } from 'react';
-import Prism from 'prismjs';
-import cn from 'classnames';
+import React, { useMemo, useRef } from 'react';
+// @ts-ignore
+import { highlightSnippet, registerDefaultLanguages } from '@ably/ui/src/core/utils/syntax-highlighter';
+// @ts-ignore
+import languagesRegistry from '@ably/ui/src/core/utils/syntax-highlighter-registry';
```

Comment:
see https://github.com/ably/voltaire/blob/main/src/types/ably-ui/index.d.ts for examples of how you could type these